io.itch.azagaya.Laigter.yaml

```diff
@@ -0,0 +1,35 @@
+app-id: io.itch.azagaya.Laigter
+runtime: org.kde.Platform
+runtime-version: "5.14"
+sdk: org.kde.Sdk
+command: laigter
+rename-icon: laigter
+rename-desktop-file: laigter.desktop
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --filesystem=home
+  - --device=dri
+
+modules:
+  - name: laigter
+    buildsystem: simple
```

Comment:
There's `qmake` build system as well.

Reply:
Thanks for the hint :slightly_smiling_face: 

I switched to the qmake build system, but it seems it's installing the icon to `/app/share/icons/hicolor/256x256/laigter.png` instead of `/app/share/icons/hicolor/256x256/apps/laigter.png`. This causes the rename-icon step to fail:

```text
Emptying app dir 'builddir'
Downloading sources
Starting build of io.itch.azagaya.Laigter
Cache hit for laigter, skipping build
Cache miss, checking out last cache hit
Cleaning up
Renaming laigter.desktop to io.itch.azagaya.Laigter.desktop
Error: icon laigter not found below /home/hugo/Documents/Git/flathub/flathub/.flatpak-builder/rofiles/rofiles-tpvTaz/files/share/icons
```